CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# dbt_stripe_source v0.10.0
```

Comment:
Can you please add CHANGELOG notes for this release

Reply:
updated